### DISCUSSION TITLE
Add source URL to image file properties post processing

### DISCUSSION THREAD

**Discussion by ALegendsTale** [Sergei-Korneev/obsidian-local-images-plus#75] [Ideas]
Hi, I have a feature request. 

Currently the source URL is replaced by the local image path. I think saving some reference of the source URL would make this plugin more feature-complete. It could simply be a property added to the output file which keeps a reference to the source it was downloaded from.

Thanks

**Comment by trinityzhub** [comment]
i have a similar opinion. But is it possible in Md link or Wiki Links?

**Comment by Sergei-Korneev** [comment]
Sorry for the late answer. That is probably a duplicate, guys
https://github.com/Sergei-Korneev/obsidian-local-images-plus/issues/36
